internal/status/advl4_status.go

```diff
@@ -326,7 +327,7 @@ func getGateways(gwNSNames []string, bulk bool, retryNum ...int) map[string]*adv
 		if err != nil {
 			utils.AviLog.Warnf("Could not get the gateway object for UpdateStatus: %s", err)
 			// retry get if request timeout
-			if strings.Contains(err.Error(), utils.K8S_ETIMEDOUT) {
```

Comment:
should we take out K8S_TIMEOUT from utils as no code is using it?

Reply:
Yeah, we don't need this. Removed.